Theme change for Hacktoberfest 2025

kitlord opened the pull request:
## Summary

- Follows brand guidelines from [guidelines](https://hacktoberfest.com/_next/static/media/brand-guidelines.13a67594.pdf)
- Updated the primary colors and background to match Hacktoberfest 2025 theme.
- Added new component for details
- Updated resource and component paths to move from react to next js
- Updated texts to follow ESLint rules
- Optimized Images for nextjs

## Why

- Aligns with Hacktoberfest 2025 branding/theme change.

## Screenshot
<img width="1563" height="778" alt="image" src="https://github.com/user-attachments/assets/e4fb8bf6-5319-43a8-b65a-707c27b617c3" />

### Notes
- Maybe we need to update the details in the hero component.

kitlord committed:
updated paths for nextjs

thet1nk3r3r commented on components/Footer.js:
```diff
@@ -0,0 +1,34 @@
+          {/* <img src={`${process.env.PUBLIC_URL}/assets/amfoss_logo.png`} alt="amfoss logo" className="logo" /> */}
+          <img src="/assets/amfoss_logo.png" alt="amfoss logo" className="logo" />
+        </div>
+        <p>We'll see you there!</p>
```
<p>{`We'll see you there!`}</p>
Writing it this way ESLint’s react/no-unescaped-entities rule is satisfied

thet1nk3r3r commented on components/Footer.js:
```diff
@@ -0,0 +1,34 @@
+        <h3>Support Open Source With</h3>
+        <div className="foss-logo">
+          {/* <img src={`${process.env.PUBLIC_URL}/assets/amfoss_logo.png`} alt="amfoss logo" className="logo" /> */}
+          <img src="/assets/amfoss_logo.png" alt="amfoss logo" className="logo" />
```
Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image`

thet1nk3r3r commented on components/Footer.js:
```diff
@@ -0,0 +1,34 @@
+        </div>
+        <p>We'll see you there!</p>
+        <div className="foot-icon">
+          <a target='_blank' href="https://hacktoberfest.com" rel="noreferrer"><img src="/assets/hacktober_logo.png" alt="hacktober logo" /></a>
```
Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image`

thet1nk3r3r commented on components/Footer.js:
```diff
@@ -0,0 +1,34 @@
+        <p>We'll see you there!</p>
+        <div className="foot-icon">
+          <a target='_blank' href="https://hacktoberfest.com" rel="noreferrer"><img src="/assets/hacktober_logo.png" alt="hacktober logo" /></a>
+          <a target='_blank' href="https://github.com/amfoss" rel="noreferrer"><img src="/assets/github.png" alt="github logo" /></a>
```
Even this change it to <Image>

thet1nk3r3r commented on components/Footer.js:
```diff
@@ -0,0 +1,34 @@
+        <div className="foot-icon">
+          <a target='_blank' href="https://hacktoberfest.com" rel="noreferrer"><img src="/assets/hacktober_logo.png" alt="hacktober logo" /></a>
+          <a target='_blank' href="https://github.com/amfoss" rel="noreferrer"><img src="/assets/github.png" alt="github logo" /></a>
+          <a target='_blank' href="https://digitalocean.com" rel="noreferrer"><img src="/assets/channels4_profile-removebg-preview 1.png" alt="digital ocean" /></a>
```
Even this change it to <Image>

thet1nk3r3r commented on components/Header/Header.js:
```diff
@@ -0,0 +1,44 @@
+    return (
+        <div className="header">
+            <div className="header-top" >
+                <a href="https://amfoss.in"><img src="/assets/amfoss_logo.png" alt="amfoss logo" className="logo" /></a>
```
Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image`

thet1nk3r3r commented on components/Header/Header.js:
```diff
@@ -0,0 +1,44 @@
+        <div className="header">
+            <div className="header-top" >
+                <a href="https://amfoss.in"><img src="/assets/amfoss_logo.png" alt="amfoss logo" className="logo" /></a>
+                <a href="https://www.amrita.edu/campus/amritapuri/"><img src="/assets/amrita_logo.png" alt="amrita logo" className="logo" /></a>
```
Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image`

thet1nk3r3r commented on components/Header/Header.js:
```diff
@@ -0,0 +1,44 @@
+                <div className="box">
+                </div>
+                <div className="left-content">
+                    <img src="/assets/HF-Horizontal-Color-Light.svg" alt="hacktoberfest logo" className="mainLogo" />
```
Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image`

thet1nk3r3r commented on components/Header/Header.js:
```diff
@@ -0,0 +1,44 @@
+                </div>
+                <div className="left-content">
+                    <img src="/assets/HF-Horizontal-Color-Light.svg" alt="hacktoberfest logo" className="mainLogo" />
+                    <p className="subtext"> It's that time of year again.<br />HacktoberFest is nearly upon us!</p>
```
<p>{`It's that time of year again.`}<br />{`HacktoberFest is nearly upon us!`}</p>

thet1nk3r3r commented on components/Prizes.js:
```diff
@@ -0,0 +1,95 @@
+'use client';
+
+import React, { useState, useEffect } from 'react';
+import { Button } from 'primereact/button';
```
its not used, eslint will be throwing errors, you can remove this line

thet1nk3r3r commented on components/Prizes.js:
```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect } from 'react';
+import { Button } from 'primereact/button';
+import { Carousel } from 'primereact/carousel';
+import { Tag } from 'primereact/tag';
```
Even tag is not used anywhere, you can remove this line

thet1nk3r3r commented on components/Prizes.js:
```diff
@@ -0,0 +1,95 @@
+    const [products, setProducts] = useState([]);
+    const [currentIndex, setCurrentIndex] = useState(0);  // Track the current center card index
+
+    const Pri = [
```
you can have the whole Pri outside the function , since its static and if you have it inside the function eslint is gonna render it everytime, wasting time and energy

thet1nk3r3r commented on components/Prizes.js:
```diff
@@ -0,0 +1,95 @@
+        return (
+            <div className={`product-card ${isCenterCard ? 'center-card' : ''}`}>
+                <div className="product-image">
+                    <img loading="lazy" src="https://images.unsplash.com/photo-1726682577615-728e4272a60c?q=80&w=2071&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D" alt={product.name} />
```
Using <img> could result in slower LCP and higher bandwidth. Consider using <Image /> from next/image

thet1nk3r3r reviewed (changes requested):
I have left few comments on the PR, kindly get them fixed
Thank you!

kitlord committed:
Fixed ESLint errors and optimized images

kitlord commented:
@ganidande905 I have fixed the ESLint errors and optimized the images and also fixed a margin issue between the footer and the body because of svg image being inline. I also removed the prize component as it seems it got pushed by accident.
